ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -170,6 +171,16 @@ gcp_connection::object_exists(const std::string &object_key, bool &exists, size_
     }
 
     std::cerr << object_key + ": " + metadata.status().message() << std::endl;
+    return get_errno(metadata.status());
+}
 
+// Maps Google Cloud Status code to an Errno value which is returned.
```

Comment:
Rather than `Errno` (which looks like you are referring to an object), I suggest simply saying "system error number`.

Reply:
Updated: https://github.com/wiredtiger/wiredtiger/commit/203c6b83eee8f9eead661087edf53e4493e4b1eb

Reply:
Has this been addressed?